pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc-view-results.adoc

```diff
@@ -0,0 +1,10 @@
+[id='view-results']
+
+.Viewing search results
+The search results are presented in a table organized into columns with the following headings:
```

Comment:
Needs an introduction sentence - what does this procedure help the user achieve?